hangman.py

```diff
@@ -194,7 +212,6 @@ def hangman(secret_word, with_hints):
     result_of_game(secret_word, letters_guessed, guesses_point)
 
 
-
 def match_with_gaps(my_word, other_word):
     '''
     my_word: string with _ characters, current guess of secret word
```

Comment:
Line 227: you can iterate not only through index but also throuhg elements it iterable objects. Simplify conditions.